sdk/monitor/azure-monitor-ingestion/README.md

```diff
@@ -0,0 +1,179 @@
+# Azure Monitor Ingestion client library for Java
+
+The Azure Monitor Ingestion client library is used to send custom logs to [Azure Monitor][azure_monitor_overview].
+
+This library allows you to send data from virtually any source to supported built-in tables or to custom tables 
+that you create in Log Analytics workspace. You can even extend the schema of built-in tables with custom columns.
+
+## Getting started
+
+### Prerequisites
+- A [Java Development Kit (JDK)][jdk_link], version 8 or later.
+- [Azure Subscription][azure_subscription]
+- A [Data Collection Endpoint][data_collection_endpoint]
+- A [Data Collection Rule][data_collection_rule]
+- A [Log Analytics workspace][log_analytics_workspace]
+
+### Include the package
+
+[//]: # ({x-version-update-start;com.azure:azure-monitor-ingestion;current})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-monitor-ingestion</artifactId>
+    <version>1.0.0-beta.1</version>
+</dependency>
+```
+
+### Create the client
+
+An authenticated client is required to upload logs to Azure Monitor. The library includes both synchronous and asynchronous forms of the clients. To authenticate, 
+the following examples use `DefaultAzureCredentialBuilder` from the [com.azure:azure-identity](https://search.maven.org/artifact/com.azure/azure-identity) package.
+
+#### Authenticating using Azure Active Directory
+You can authenticate with Azure Active Directory using the [Azure Identity library][azure_identity].
+To use the [DefaultAzureCredential][DefaultAzureCredential] provider shown below, or other credential providers provided with the Azure SDK, please include the `azure-identity` package:
+[//]: # ({x-version-update-start;com.azure:azure-identity;dependency})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-identity</artifactId>
+    <version>1.5.2</version>
+</dependency>
+```
+Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables: AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET.
+
+#### Synchronous Logs Ingestion client
+
+```java readme-sample-createLogsIngestionClient
+DefaultAzureCredential tokenCredential = new DefaultAzureCredentialBuilder().build();
+
+LogsIngestionClient client = new LogsIngestionClientBuilder()
+        .endpoint("<data-collection-endpoint>")
+        .credential(tokenCredential)
+        .buildClient();
+```
+
+#### Asynchronous Logs Ingestion client
+
+```java readme-sample-createLogsIngestionAsyncClient
+DefaultAzureCredential tokenCredential = new DefaultAzureCredentialBuilder().build();
+
+LogsIngestionAsyncClient asyncClient = new LogsIngestionClientBuilder()
+        .endpoint("<data-collection-endpoint>")
+        .credential(tokenCredential)
+        .buildAsyncClient();
+```
+### Uploading logs
+
+For examples on how to upload logs, see the [Examples](#examples) section.
```

Comment:
Is this section needed? It feels abrupt and just jumps to somewhere else in the README. Should we think about doing table of contents at the top of the README instead which would allow for someone to quick jump to samples?